src/components/SearchField.jsx

```diff
@@ -137,6 +137,7 @@ class SearchField extends React.Component {
         <button
           className="react-search-field-button"
           type="button"
+          aria-label="search-btn"
```

Comment:
According to [ARIA14](https://www.w3.org/WAI/WCAG20/Techniques/ua-notes/aria#ARIA14), it would be recommended if we put a descriptive label. e:g: `search button` instead of `search-btn`

Reply:
Good catch, will update now